Resources/ArmTemplates/template.json

```diff
@@ -98,15 +98,38 @@
         "servicebusName": "[concat(variables('ResourcePrefix'), '-ns')]",
         "servicebusSku": "[if(variables('productionEnvironment'), 'Premium', 'Standard')]",
         "storageAccountName": "[tolower(concat('dfc', parameters('environment'), 'appsharedstr'))]",
-        "cmsTopicName": "cms-messages",
-        "cmsTopicName-draft": "[concat(variables('cmsTopicName'),'-draft')]",
         "apimJobProfilesProductDisplayName": "Job Profiles",
         "addDevelopersGroupToJPProduct": "[equals(tolower(parameters('environment')), 'oat')]",
         "MonitoringAppServicePlanName": "[concat(variables('ResourcePrefix'), '-mon-asp')]",
         "MonitoringAppInsightsName": "[concat(variables('ResourcePrefix'), '-mon-ai')]",
         "MonitoringFunctionAppName": "[concat(variables('ResourcePrefix'), '-mon-fa')]",
-        "ActionGroupName": "[concat(variables('ResourcePrefix'), '-actgrp')]"
+        "ActionGroupName": "[concat(variables('ResourcePrefix'), '-actgrp')]",
+        "TopicsToCreate": [
+            "cms-messages",
+            "cms-messages-draft",
+            "job-profile-refresh",
+            "job-profile-refresh-draft"
+        ]
     },
+    "functions": [
+        {
+            "namespace": "dfc",
+            "members": {
+                "truncateDeploymentName": {
+                    "parameters": [
+                        {
+                            "name": "deploymentName",
+                            "type": "string"
+                        }
+                    ],
+                    "output": {
+                        "type": "string",
+                        "value": "[skip(parameters('deploymentName'), sub(length(parameters('deploymentName')),64))]"
```

Comment:
Where does deploymentName come from?  It's not in the parameters list

Reply:
The deploymentName parameter is defined in the "parameters" block of the function, and is scoped locally to the function.